sky/clouds/service_catalog/data_fetchers/fetch_fluidstack_cloud.py

```diff
@@ -0,0 +1,139 @@
+"""A script that generates the Lambda Cloud catalog.
+
+Usage:
+    python fetch_lambda_cloud.py [-h] [--api-key API_KEY]
+                                 [--api-key-path API_KEY_PATH]
+
+If neither --api-key nor --api-key-path are provided, this script will parse
+`~/.lambda/lambda_keys` to look for Lambda API key.
+"""
+import argparse
+import csv
+import json
+import os
+import requests
+from typing import List
+from sky.clouds.service_catalog import constants
+
+ENDPOINT = 'http://console.fluidstack.io/api/plans'
+DEFAULT_FLUIDSTACK_API_KEY_PATH = os.path.expanduser(
+    '~/.fluidstack/fluidstack_api_key')
+DEFAULT_FLUIDSTACK_API_TOKEN_PATH = os.path.expanduser(
+    '~/.fluidstack/fluidstack_api_token')
+
+GPU_MAP = {
+    'A100_PCIE_40GB': 'A100',
+    'T4': 'T4',
+    'Tesla_V100_PCIE': 'V100',
+    'A10': 'A10',
+    'A100_PCIE_80GB': 'A100-80GB',
+    'Quadro_RTX_6000_16GB': 'RTX6000',
+    'RTX_A4500_16GB': 'A4500',
+    'RTX_3060_Ti': 'RTX3060Ti',
+    'Quadro_RTX_4000_12GB': 'RTX4000',
+    'Quadro_P5000_12GB': 'P5000',
+    'RTX_A4000': 'A4000',
+    'Quadro_RTX_5000_16GB': 'RTX5000',
+    'A30': 'A30',
+    'A40': 'A40',
+    'RTX_3090': 'RTX3090',
+    'RTX_A6000': 'A6000',
+    'RTX_3080': 'RTX3080',
+    'RTX_A5000': 'A5000',
+    'A100_SXM4_80GB': 'A100-80GB',
+    'A100_NVLINK_80GB': 'A100-80GB-NVLink',
+    'A100_NVLINK': 'A100_NVLINK'
```

Comment:
Let's avoid adding the `NVLink` to the accelerator name, but just keep them `A100` and `A100-80GB`. Our higher level code will handle the case when multiple instance types are involved for the same accelerator, hinting the user to explicitly specify the instance type.